index.js

```diff
@@ -0,0 +1,71 @@
+import BookStore from './modules/books.js';
+import { loadBooksCollection, loadContacts } from './modules/booksFunction.js';
+import getCurrentDate from './modules/CurrentDate.js';
+
+const newBookStorage = new BookStore();
+
+const currentDateElement = document.querySelector('.datetime');
+currentDateElement.textContent += getCurrentDate();
```

Comment:
After installing all dependencies and running your project, your webpage navigation is not working and the current date is not being displayed. Please investigate this bug and try to resolve it👍